tests/E2E/docker-compose.yml

```diff
@@ -24,6 +24,7 @@ services:
       DB_SERVER: db
       PS_INSTALL_AUTO: 1
       PS_COUNTRY: fr
+      PS_LANGUAGE: en
```

Comment:
Does this actually fixes the issue? This value is already set by default: https://github.com/PrestaShop/docker/blob/c93fb39ae49bc8b4720577c0a6df8631e12eecfb/base/images/7.2-apache/Dockerfile#L13

Reply:
I had the same reflexion we had this (with @boubkerbribri ) to try if it's ok during the nightly

Reply:
@PierreRambaud @Quetzacoalt91 I'm not sure about that.
You can have a look at the nightly report test full/08_stocks/4_update_product_quantity.js :+1: 
We have this error 
``` AssertionError: expected 'Please enter a valid value. The two nearest valid values are 10 and 11.' to include 'Veuillez saisir une valeur valide.' ```
at /prestashop-core-nightly/reports/2019-06-05-1.7.6.x.html#040f4062-063a-4d51-8514-24e79b41efeb

Reply:
We'll investigate to understand clearly what's happening :/

Reply:
I'm not saying the shop is properly in English, I'm saying this change won't fix your issue. :)

Reply:
> I'm not saying the shop is properly in English, I'm saying this change won't fix your issue. :)
Sorry for misunderstanding.